[2024-02-26T22:34:20.425+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-26T22:34:04.498004+00:00 [queued]>
[2024-02-26T22:34:20.438+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-26T22:34:04.498004+00:00 [queued]>
[2024-02-26T22:34:20.439+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-26T22:34:20.460+0000] {taskinstance.py:2191} INFO - Executing <Task(SnowflakeOperator): create_snowflake_schema> on 2024-02-26 22:34:04.498004+00:00
[2024-02-26T22:34:20.468+0000] {standard_task_runner.py:60} INFO - Started process 70 to run task
[2024-02-26T22:34:20.474+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'create_snowflake_schema', 'manual__2024-02-26T22:34:04.498004+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmp2g5l4kt3']
[2024-02-26T22:34:20.477+0000] {standard_task_runner.py:88} INFO - Job 46: Subtask create_snowflake_schema
[2024-02-26T22:34:20.539+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-26T22:34:04.498004+00:00 [running]> on host 5d25474dd79a
[2024-02-26T22:34:20.651+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='create_snowflake_schema' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T22:34:04.498004+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T22:34:04.498004+00:00'
[2024-02-26T22:34:20.653+0000] {sql.py:276} INFO - Executing: 
                    CREATE SCHEMA IF NOT EXISTS Weather ;
            
[2024-02-26T22:34:20.670+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-26T22:34:20.684+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-26T22:34:20.685+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.34
[2024-02-26T22:34:20.687+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-02-26T22:34:21.043+0000] {cursor.py:1028} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-02-26T22:34:21.128+0000] {cursor.py:1041} INFO - query execution done
[2024-02-26T22:34:21.130+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-26T22:34:21.131+0000] {sql.py:450} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS Weather ;, parameters: None
[2024-02-26T22:34:21.133+0000] {cursor.py:1028} INFO - query: [CREATE SCHEMA IF NOT EXISTS Weather ;]
[2024-02-26T22:34:21.204+0000] {cursor.py:1041} INFO - query execution done
[2024-02-26T22:34:21.205+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-26T22:34:21.206+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-26T22:34:21.207+0000] {snowflake.py:400} INFO - Rows affected: 1
[2024-02-26T22:34:21.208+0000] {snowflake.py:401} INFO - Snowflake query id: 01b29e2a-3201-0126-0000-000805ce5101
[2024-02-26T22:34:21.209+0000] {cursor.py:1028} INFO - query: [COMMIT]
[2024-02-26T22:34:21.284+0000] {cursor.py:1041} INFO - query execution done
[2024-02-26T22:34:21.285+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-26T22:34:21.286+0000] {connection.py:718} INFO - closed
[2024-02-26T22:34:21.351+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2024-02-26T22:34:21.441+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=create_snowflake_schema, execution_date=20240226T223404, start_date=20240226T223420, end_date=20240226T223421
[2024-02-26T22:34:21.493+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-26T22:34:21.557+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
